e2e/libs/k8s/k8s.py

```diff
@@ -80,6 +80,15 @@ def wait_all_pods_evicted(node_name):
 
     assert evicted, 'failed to evict pods'
 
+def get_node_status(node_name):
```

Comment:
```suggestion
def is_node_ready(node_name):
```
The function name should reflect its purpose, which here checks for the node `Ready` condition. A typical naming convention for boolean-returning functions is to prefix with `is_`, `has_`, etc.

Reply:
Update